src/kryptos/api/app.py

```
@@ -52,6 +53,19 @@ def create_app(index: ArtifactIndex | None = None) -> FastAPI:
     install_log_streaming()
 
     app = FastAPI(title="Kryptos API", version="0.1.0")
+
+    # CORS: allow the Netlify-hosted SPA to call this backend cross-origin.
+    # Configure via KRYPTOS_CORS_ORIGINS (comma-separated); defaults to the
+    # production Netlify site. "*" disables origin checks (dev only).
+    cors_origins = os.environ.get("KRYPTOS_CORS_ORIGINS", "https://kryptos-k4.netlify.app")
```

Pylint (4.0.7)

[convention] Line too long (91/88)

(C0301)